fixture/app.js

```diff
@@ -9,16 +9,68 @@
 const http = require('http')
 const express = require('express')
 const bodyParser = require('body-parser')
+const path = require('path')
+const ejs = require('ejs')
+const urllib = require('urllib')
+const url = require('url')
+var qs = require('querystring')
```

Comment:
Hey, it's better use `const` here. All the required modules better use `const` variable. Since we use `use strict` and `node >= 6.0.0`.